File: app/src/main/java/com/mouredev/weeklychallenge2022/Challenge31_test.js
const returnLeapYear = require("./Challenge31");

const test = { input: [4, 20, 1800, 1900, 2100, 2200, 2300, 2500, 2000, 2400] };

test.input.forEach((element) => {
  console.log(`Year: ${element}: \n ${returnLeapYear.func(element)}`);
});

File: app/src/main/java/com/mouredev/weeklychallenge2022/Challenge31_test.rb
# frozen_string_literal: true

require_relative './Challenge31'

test = { input: [4, 20, 1800, 1900, 2100, 2200, 2300, 2500, 2000, 2400] }

test[:input].each do |year|
  puts "Year input: #{year}"
  year_to_test = Main.new(year)
  puts year_to_test.execute
end
